README.md

```diff
@@ -439,6 +439,26 @@ get "/" do
 end
 ```
 
+### log_level
+You can specify log level.
+
+```ruby
+Kiev.configure do |config|
+  # One of: 0, 1, 2, 3, 4 (DEBUG, INFO, WARN, ERROR, FATAL)
+  config.log_level = 0
+end
+```
+
+### enable_filter_for_log_levels
```

Comment:
Let's add a piece of information about a default behavior, WDYT?
To reflect that if no `enable_filter_for_log_levels` set then the filtering will be applied to all levels.